src/plugin/regions/region.js

```diff
@@ -399,6 +435,15 @@ export class Region {
             this.fireEvent('contextmenu', e);
             this.wavesurfer.fireEvent('region-contextmenu', this, e);
         });
+        /* Edit content */
+        if (this.contentEditable){
+            this.contentEl.addEventListener('blur', this.onContentEdit.bind(this));
+            this.contentEl.addEventListener('click', e => e.stopPropagation());
+        }
+        /* Edit content */
+        if (this.removeButton){
+            this.removeButtonEl.addEventListener('click', this.onRemove.bind(this));
```

Comment:
every addEventListener needs a removeEventListener.

Reply:
sorry, I don't see examples in code. 
For `this.element.addEventListener` I don't see any `removeEventListener`.

Reply:
would it be ok?:
```js
        /* Edit content */
        if (this.contentEditable){
            this.contentEl.addEventListener('blur', this.onContentBlur.bind(this));
            this.contentEl.addEventListener('click', this.onContentClick.bind(this));
        }
        /* Remove button */
        if (this.removeButton){
            this.removeButtonEl.addEventListener('click', this.onRemove.bind(this));
        }

        this.on('remove', ()=>{
            this.contentEl.removeEventListener('blur', this.onContentBlur.bind(this));
            this.contentEl.removeEventListener('click', this.onContentClick.bind(this));
            this.removeButtonEl.removeEventListener('click', this.onRemove.bind(this));
        });
```

Reply:
the idea is when you call the plugin's `destroy()` method, you can be sure all event listeners are cleaned up.